source/routing/raptor_solution_reader.cpp

```diff
@@ -88,7 +88,14 @@ get_out_st_dt(const std::pair<const type::StopTime*, DateTime>& in_st_dt,
     return {nullptr, 0};
 }
 
-bool is_valid(const Journey& j) {
+template<typename Visitor>
+bool is_valid(const RaptorSolutionReader<Visitor>& reader, const Journey& j) {
```

Comment:
oops, sorry, my bad. And by reading other functions accepting `reader` as argument, i'm ok with that